File: src/cz/vutbr/fit/interlockSim/context/EmptyContextException.java
/* Brno University of Technology
 * Faculty of Information Technology
 * 
 * BSc Thesis	2006/2007
 * 
 * Railway Interlocking Simulator
 * 
 * Bedrich Hovorka
 */
package cz.vutbr.fit.interlockSim.context;

/**
 * In context be missing something
 */
public class EmptyContextException extends Exception {
	/**
	 * @see Exception#Exception()
	 */
	public EmptyContextException() {
		//EMPTY
	}

	/**
	 * @see Exception#Exception(String)
	 */
	public EmptyContextException(String message) {
		super(message);
	}

	/**
	 * @see Exception#Exception(Throwable)
	 */
	public EmptyContextException(Throwable cause) {
		super(cause);
	}

	/**
	 * @see Exception#Exception(String, Throwable)
	 */
	public EmptyContextException(String message, Throwable cause) {
		super(message, cause);
	}
}

File: src/main/kotlin/cz/vutbr/fit/interlockSim/context/EmptyContextException.kt
/* Brno University of Technology
 * Faculty of Information Technology
 *
 * BSc Thesis  2006/2007
 *
 * Railway Interlocking Simulator
 *
 * Bedrich Hovorka
 */
package cz.vutbr.fit.interlockSim.context

/**
 * In context be missing something
 */
class EmptyContextException : Exception {
	/**
	 * @see Exception.Exception
	 */
	constructor() : super() {
		// EMPTY
	}

	/**
	 * @see Exception.Exception
	 */
	constructor(message: String) : super(message)

	/**
	 * @see Exception.Exception
	 */
	constructor(cause: Throwable) : super(cause)

	/**
	 * @see Exception.Exception
	 */
	constructor(message: String, cause: Throwable) : super(message, cause)
}
